.github/workflows/ci.yml

```diff
@@ -352,32 +352,39 @@ jobs:
     continue-on-error: true
     if: contains(github.head_ref, 'static') || contains(github.ref, 'release-') || github.event_name == 'schedule'
     runs-on: ubuntu-20.04
+    container: alpine:edge
     steps:
-    - uses: actions/checkout@v2
-      with:
-          submodules: recursive
+    - name: Install dependencies
+      run: apk add --no-cache git gcc g++ linux-headers cmake wget unzip py3-wheel py3-setuptools py3-pip meson ninja
+    - name: Checkout rizin
+      run: |
+        git clone https://github.com/${{ github.repository }}
+        cd rizin
+        git fetch origin ${{ github.ref }}
+        git checkout -b local_branch FETCH_HEAD
+        git submodule init
+        git submodule update
     - name: Checkout our Testsuite Binaries
-      uses: actions/checkout@v2
-      with:
-          repository: rizinorg/rizin-testbins
-          path: test/bins
-    - name: Install static
+      run: git clone https://github.com/rizinorg/rizin-testbins test/bins
+    - name: Compile with meson
       run: |
-        ./sys/static.sh
-        sudo make symstall
+        CFLAGS="-static" LDFLAGS="-static" meson --prefix=${HOME}/.local --buildtype release --default-library static build
+        ninja -C build && ninja -C build install
+      working-directory: rizin
     - name: Run tests
       run: |
+        export PATH=${HOME}/.local/bin:${PATH}
         rizin -v
         rz-test -v
         cd test
         make
```

Comment:
replace this with unit tests + rz-test call. No `make` because it is going to be killed.